### ISSUE TITLE
[Feature Request] Ability to monitor audio inputs directly

### ISSUE THREAD

**Issue by Phr3d13** [milkdrop2077/MilkDrop3#71]
Another feature request. Title is self explanatory.

**Comment by milkdrop2077** [comment]
I'm not sure what you mean

**Comment by Phr3d13** [comment]
The only way I can get milkdrop to listen to an audio input ( line level input of my scarlett 4i4) is to use a second program called VB-Cable. In the audio picker, there are only audio outputs.

**Comment by milkdrop2077** [comment]
ok, well the wasapi loopback capture code is something quite hard to update. I've been trying to get some help from some other ++skilled developers and no ones seems to have the skills or time to help with that.

**Comment by Phr3d13** [comment]
Is what we have here on github even buildable? If so, what are you using to compile? Barely know what I'm doing to try to help, so any help you could give me to help you would be helpful...

**Comment by milkdrop2077** [comment]
Yes, it's buildable. Microsoft Visual Studio. Again, the audio part is really technical. You need advanced C++ coding skills and need to know about audio and the Windows API.

**Comment by Phr3d13** [comment]
any chance you could give me current code and not 1 year old code?

**Comment by Phr3d13** [comment]
Yeah, it must be over my head. Can't even get it to compile. But then again, I'm used to VSC and not VS (lots of little arduino type projects). Never went to school for any sort of coding/programming, Can't even figure out how to tackle all the missing #indlude(d) files.
